Makefile

```diff
@@ -20,11 +20,11 @@
 #==================================================================================================
 
 
-CPP                =
+CPP                = 
 PYTHON             = python
 COMPILER_MODE      = DEBUG
 PRECISION          = DOUBLE
-OPENMP             = 1
+OPENMP             = 0
```

Comment:
I agree that we should have some defaults and stick to them, but I'd say that I don't know what they are since we've been changing them all the time...

I took this opportunity to make a sorely needed change, with is to make it possible to specify MPI=1 in the main makefile so that those people who have the mpi library installed, but not the compiler wrappers can still compile with mpi easily. Since this already happens on cosmos, it's bound to happen elsewhere also.